### ISSUE TITLE
Play store

### ISSUE THREAD

**Issue by FLYBYME** [maciek252/STorM32_GUI_Android#1]
Is this app up on the play store?

**Comment by maciek252** [comment]
No, too early for this. First I'll need to prepare a test version, which will compile and display scripts but without the wizards. I'll try to do this in this year. Of course any help will make this time shorter. Earlier I'll prepare an upgraded APK corresponding to the current state of the development.

Yet another problem is that there are still some changes in the controller, but maybe sticking to the version 0.96 will be enough, just for the beginning.
